cluster-autoscaler/cloudprovider/exoscale/OWNERS

```diff
@@ -0,0 +1,4 @@
+approvers:
```

Comment:
The mentioned people are not in K8S org. They will not pass validation. Rename OWNERS to OWNERS-unofficial so that we know who to contact and we pass the org-imposed validation.

Reply:
Done

Reply:
Sorry, you must have read the previous version of the comment. I edited it shortly after posting.

Reply:
I did indeed, my fault, have updated again.